src/patternfly/components/MenuToggle/menu-toggle.scss

```diff
@@ -10,10 +10,10 @@
 
 :where(:root, .#{$menu-toggle}) {
   --#{$menu-toggle}--ColumnGap: var(--pf-t--global--spacer--sm);
-  --#{$menu-toggle}--PaddingBlockStart: var(--pf-t--global--spacer--sm);
-  --#{$menu-toggle}--PaddingInlineEnd: var(--pf-t--global--spacer--sm);
-  --#{$menu-toggle}--PaddingBlockEnd: var(--pf-t--global--spacer--sm);
-  --#{$menu-toggle}--PaddingInlineStart: var(--pf-t--global--spacer--sm);
+  --#{$menu-toggle}--PaddingBlockStart: var(--pf-t--global--spacer--control--vertical--default);
+  --#{$menu-toggle}--PaddingInlineEnd: var(--pf-t--global--spacer--action--horizontal--default);
+  --#{$menu-toggle}--PaddingBlockEnd: var(--pf-t--global--spacer--control--vertical--default);
+  --#{$menu-toggle}--PaddingInlineStart: var(--pf-t--global--spacer--action--horizontal--default);
```

Comment:
Looks like these are getting action tokens on the L/R. It should use the control tokens for the inline padding. The only menu toggles that should get the action tokens for the inline padding are the ones that are supposed to look like buttons!